src/main/kotlin/calculator/StringCalculatorCounter.kt

```diff
@@ -0,0 +1,53 @@
+package calculator
+
+import java.util.*
+
+class StringCalculatorCounter(private val formula: List<String>) {
+    init {
+        if (formula.isEmpty()){
+            throw IllegalArgumentException("Please Check the Input")
+        }
+        for ((index, element) in formula.withIndex()) {
+            validateNumber(index, element)
+            validateSymbol(index, element)
+        }
+
+    }
+    fun validateNumber(index : Int, element : String){
+        if (index % 2 == 0 && !element.matches(Regex("^\\d+$"))){
+            throw IllegalArgumentException("Please Check the Input")
+        }
+    }
+
+    fun validateSymbol(index : Int, element: String){
+        if (index % 2 == 1 && !element.matches(Regex("^[+/\\-*]+$"))){
+            throw IllegalArgumentException("Please Check the Input")
+        }
+    }
+
+
+    fun calculate(): Double {
+        val basicOperationsCalculator = BasicOperationsCalculator()
+        val queue: Queue<String> = LinkedList(formula)
+        var curNum = queue.poll().toDouble()
+
+        while (queue.isNotEmpty()) {
+            val symbol = queue.poll()
+            val num = queue.poll().toDouble()
+            curNum = basicOperationsCalculator.operation(curNum, num, symbol)
+        }
+
+        return curNum
+    }
+
+    constructor(input: String) : this(input.split(
+        " "))
```

Comment:
아래 처럼 코드를 정렬해보는 건 어떨까요?
```suggestion
    constructor(input: String) : this(input.split(" "))
```